py3status/modules/insync.py

```diff
@@ -85,14 +87,17 @@ def insync(self):
             }
 
         color = self.py3.COLOR_DEGRADED
+        format = self.format
         if status == "Insync doesn't seem to be running. Start it first.":
             color = self.py3.COLOR_BAD
-            status = STRING_ERROR
+            format = STRING_ERROR
```

Comment:
one option would be to use `self.py3.error()` here so it is treated as an error rather than as output.